### PR TITLE
WM4- Kavita-patil- Full Stack Project Assessment

### PR DESCRIPTION
**Pull request by kavitappatil**

### CODE REVIEW

File: client/src/buttons/LikeIcon.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+function LikeIcon({ onThumbsUp }) {
+  return (
+    <button
```
Comment (SallyMcGrath) [inline client/src/buttons/LikeIcon.js:5]: What does this button do? How can a non-sighted user find out? When you ran Lighthouse, what did the report say and what steps did you take to address it?

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
This is a good start @kavitappatil. Thanks for your work on this. I think there's still some work to be done here. I think there are some specific requirements you have missed. 

I cannot see your Lighthouse report. When I ran Lighthouse on your code, it scored 68/100, which is very low. This seems super fixable. What will you do?

Take a look and have a think. Good start! 👍

File: client/src/VideoCard.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import DeleteButton from "./buttons/DeleteButton";
+import LikeIcon from "./buttons/LikeIcon";
+import DislikeIcon from "./buttons/DislikeIcon";
+import "./videoCard.css";
+
+function VideoCard({ video, handleDelete, onThumbsDown }) {
+  const url = video.url.replace("watch?v=", "embed/");
+  const [voteCount, setVoteCount] = useState(video.rating);
+
+   function likeVote() {
```
[the remark this answers is not in the document]
Comment (kavitappatil) [reply client/src/VideoCard.js:11]: I have used the inline arrow function for likeVote.

File: client/src/VideoCard.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    <div className="card">
+      <iframe
+        src={url}
+        frameborder="0"
+        allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+      ></iframe>
+      <div className="card-body">
+        <h5 className="card-title">{video.title}</h5>
+        {/* <p className="card-text">
```
[the remark this answers is not in the document]
Comment (kavitappatil) [reply client/src/VideoCard.js]: removed as JSON data has only a title.

File: client/src/VideoCard.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+          Some quick example text to build on the card title and make up the
```
[the remark this answers is not in the document]
Comment (kavitappatil) [reply client/src/VideoCard.js]: removed as JSON data has only a title.

File: client/src/VideoCard.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+          bulk of the card's content.
```
[the remark this answers is not in the document]
Comment (kavitappatil) [reply client/src/VideoCard.js]: removed as JSON data has only a title.

File: client/src/VideoCard.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+        </p> */}
```
[the remark this answers is not in the document]
Comment (kavitappatil) [reply client/src/VideoCard.js]: removed as JSON data has only a title.

File: client/src/videoCard.css [hunk 7]
```diff
@@ -0,0 +1,9 @@
+@media only screen and (min-width: 768px) {
+    iframe {
```
[the remark this answers is not in the document]
Comment (kavitappatil) [reply client/src/videoCard.css:2]: Change CSS to use "%" as will occupy space on the card.

File: client/src/VideoCard.js [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+    if (count < 0){
+        count = 0
+    }
+    setVoteCount(count);
+  }
+
+  return (
+    <div className="card">
+      <iframe
+        src={url}
```
[the remark this answers is not in the document]
Comment (kavitappatil) [reply client/src/VideoCard.js:23]: yes, I updated the title and missing allow attributes. I have set the height and width in CSS.

File: client/src/buttons/DislikeIcon.js [hunk 9]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+function DislikeIcon({onThumbsDown}) {
+  return (
+    <button
```
[the remark this answers is not in the document]
Comment (kavitappatil) [reply client/src/buttons/DislikeIcon.js:5]: aria-label added to the dislike button.

File: client/src/buttons/DeleteButton.js [hunk 10]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+function DeleteButton({onDelete }) {
+  return (
+    <button
+      className="p-2 mb-1 btn btn-sm btn-outline-danger"
+      type="button"
```
[the remark this answers is not in the document]
Comment (kavitappatil) [reply client/src/buttons/DeleteButton.js:7]: aria-label added to the delet button.

File: client/src/buttons/LikeIcon.js (hunk 1 above)
Comment (kavitappatil) [reply client/src/buttons/LikeIcon.js:5]: aria-label added to the like button.

Comment (kavitappatil) [reply client/src/buttons/LikeIcon.js:5]: Hi Sally, thank you so much for reviewing my code, I really appreciate your effort.

File: client/src/VideoCard.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (kavitappatil) [reply client/src/VideoCard.js]: Screenshot 2023-01-27 at 23 30 50

attached lighthouse report - accessibility is 100% now.

**Comment by SallyMcGrath** [comment]
⭐

**Comment by kavitappatil** [comment]
https://cyf-videos-kp-react.onrender.com

**Pull request closed without merging**